app/client/api.ts

```diff
@@ -151,7 +151,6 @@ export function getHeaders() {
   const accessStore = useAccessStore.getState();
   const headers: Record<string, string> = {
     "Content-Type": "application/json",
-    "x-requested-with": "XMLHttpRequest",
```

Comment:
Tested works fine with OpenAI & Gemini API, do you have any other suggestions?

Reply:
> Tested works fine with OpenAI & Gemini API, do you have any other suggestions?

@fred-bf 

The one-api works fine with both OpenAI (gpt,dalle) and Gemini. However, I haven't been able to test other models yet because I don't have access to them.

Reply:
Because both the OpenAI and Google Gemini APIs return data in JSON format, this field is not expected to be necessary for the current scenario.

Reply:
The origin commit aims to resolve not showing auth page, I just double check it works as expected when server returns 401 status code.

https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/commit/600b1814a1b982e6faca151afff0518b15884c79

Reply:
> The origin commit aims to resolve not showing auth page, I just double check it works as expected when server returns 401 status code.
> 
> [600b181](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/commit/600b1814a1b982e6faca151afff0518b15884c79)

It's better to keep I think